test/JDBC/input/storedProcedures/Test-sp_helpsrvrolemember-14_4-vu-prepare.sql

```diff
@@ -0,0 +1,29 @@
+CREATE PROC test_sp_helpsrvrolemember_proc @rolename AS sys.SYSNAME = NULL
```

Comment:
Let us not use 14_4 as tag, instead use something like Test-sp_helpsrvrolemember-dep-vu-prepare.sql maybe